components/server/src/workspace/headless-log-controller.ts

```diff
@@ -76,9 +87,9 @@ export class HeadlessLogController {
                 // [gpl] It's a bit sad that we have to duplicate this access check... but that's due to the way our API code is written
                 const resourceGuard = new CompositeResourceAccessGuard([
                     new OwnerResourceGuard(user.id),
-                    new WorkspaceLogAccessGuard(user, this.hostContextProvider),
+                    new TeamMemberResourceGuard(user.id)
```

Comment:
Don't we want to also keep the `WorkspaceLogAccessGuard` here (as in `websocket-connection-manager.ts`) for "legacy mode"?